flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -1054,4 +1075,13 @@ public T getSerializer() {
 			return Collections.emptyMap();
 		}
 	}
+
+	/**
+	 * The clean level determines whether need to clean or whether need to clean recursively.
+	 */
+	public enum CLEANLEVEL {
```

Comment:
This should be in line with regular java style, i.e. the name should be something like `ClosureCleanerLevel` or `ClosureCleanerConfig`. The individual options could be something like `NONE`, `TOP_LEVEL`, or `RECURSIVE`, with javadoc that describes what the options do inserted here.